README.md

```diff
@@ -21,6 +21,14 @@ git clone git@github.com:lagom/online-auction-java.git
 cd online-auction-java
 sbt runAll
 ```
+## Importing into IDEs
+
+### Intellij IDEA
+* **[Follow steps given here](http://www.lagomframework.com/documentation/1.3.x/java/IntellijSbtJava.html)**
+* Add **[Lombok](https://github.com/mplushnikov/lombok-intellij-plugin)** plugin by going to `File -> Settings -> Plugins` 
+
+### Eclipse
+* **[Follow steps given here](http://www.lagomframework.com/documentation/1.3.x/java/EclipseSbt.html)**
```

Comment:
Hi @lakhina , it would be good if we avoid this kind of wording. The readme, and specially the steps to run for the first time or import into IDE, are the first things a new user will see.

How about something like: "to import the `online-auction-java`project into Eclipse you can have a look at [the Lagom Documentation](http://www.lagomframework.com/documentation/1.3.x/java/EclipseSbt.html)."

Same thing applies for when importing the project in IntelliJ